src/ui/panels/quests.js

```diff
@@ -62,6 +63,13 @@ export class Quests {
                 }
             });
 
+        this.questPointsText = dashboard.add
+            .text(645, 233, String(characterData.getQuestPoints()), {
```

Comment:
I think it auto casts to String but you can double check

Reply:
Removed the cast